opm/input/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -2380,14 +2587,25 @@ std::vector<double> EclipseGrid::createDVector(const std::array<int,3>& dims, st
 
 namespace Opm {
     EclipseGridLGR::EclipseGridLGR(const std::string& self_label, const std::string& father_label_, 
-                                   int father_lgr_level, std::size_t nx, std::size_t ny, 
-                                   std::size_t nz, const vec_size_t& father_lgr_index)
+                                   std::size_t nx, std::size_t ny, std::size_t nz, 
+                                   vec_size_t father_lgr_index, const std::array<int,3>& low_fahterIJK_, 
+                                   const std::array<int,3>& up_fahterIJK_)
```

Comment:
The `low_fatherIJK_` and `up_fatherIJK_` parameters are unused in this constructor.  Is that intentional?

Reply:
this will be used in the future for the localization algorithm based on the multiplicity.